.github/workflows/container.yml

```diff
@@ -0,0 +1,39 @@
+name: Container Check
+
+on:
+  push:
+    paths-ignore:
+      - '**/*.md'
+  pull_request:
+    paths-ignore:
+      - '**/*.md'
+
+jobs:
+  container-check:
+    runs-on: ubuntu-latest
+    if: (github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository)
+    steps:
+      - name: Clone the code
+        uses: actions/checkout@v4
+
+      - name: Setup Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: '~1.22'
+
+      - name: Prepare project-v4
+        working-directory: testdata/project-v4
```

Comment:
could run a matrix here to check all 3 testdata folders (project-v4,  project-v4-multigroup  and project-v4-with-plugins). Needed or just one is enough?